src/gc-pages.c

```diff
@@ -180,6 +182,7 @@ void jl_gc_free_page(jl_gc_pagemeta_t *pg) JL_NOTSAFEPOINT
     madvise(p, decommit_size, MADV_DONTNEED);
 #endif
     msan_unpoison(p, decommit_size);
+    jl_atomic_fetch_add_relaxed(&gc_heap_stats.bytes_decomitted, decommit_size);
```

Comment:
This seems like sort of a weird place to track this, since it only counts the pages we return to the OS, but ignores the pages we prepare for immediate reuse

Reply:
Right, so that variable tracks specifically pages returned to the OS, the one that tracks the returned to reuse is `bytes_freed`. I couldn't find a better name so up for suggestions